packages/js/product-editor/src/constants.ts

```diff
@@ -1,3 +1,44 @@
 export const NUMBERS_AND_ALLOWED_CHARS = '[^-0-9%s1%s2]';
 export const NUMBERS_AND_DECIMAL_SEPARATOR = '[^-\\d\\%s]+';
 export const ONLY_ONE_DECIMAL_SEPARATOR = '[%s](?=%s*[%s])';
+
+// This should never be a real slug value of any existing shipping class
+export const ADD_NEW_SHIPPING_CLASS_OPTION_VALUE =
+	'__ADD_NEW_SHIPPING_CLASS_OPTION__';
```

Comment:
What happens if it is?

Reply:
No idea! That comment was copied over from the `/client/products/constants.ts` file with the constants themselves. It looks like it was originally added in https://github.com/woocommerce/woocommerce/pull/35123, so we'd have to ask @mdperez86 .

Reply:
Ah, got it. I suppose that it is unlikely for somebody manually type in such a name, so I'm okay with leaving this as-is for now.